app/screens/LoginScreen.tsx

```diff
@@ -35,72 +34,81 @@ export const LoginScreen = () => {
       .catch(() => alert("Wrong Email or Password"));
   };
 
+  const getThemeStyles = () => {
```

Comment:
- use StyleSheet.create for creating styles
- don't use color literals, use constants from theme instead
- it's better to define that function outside the component